README.md

```diff
@@ -15,24 +15,15 @@ The SDK provides one crate per AWS service. You must add [Tokio](https://crates.
 
 ```toml
 [dependencies]
-aws-sdk-dynamodb = { git = "https://github.com/awslabs/aws-sdk-rust", tag = "v0.0.16-alpha", package = "aws-sdk-dynamodb" }
+aws-config = { git = "https://github.com/awslabs/aws-sdk-rust", tag = "v0.0.17-alpha", package = "aws-config" }
+aws-sdk-dynamodb = { git = "https://github.com/awslabs/aws-sdk-rust", tag = "v0.0.17-alpha", package = "aws-sdk-dynamodb" }
 tokio = { version = "1", features = ["full"] }
 ```
-3. Provide your AWS credentials as environment variables:
-  > **Note:** The alpha version of the SDK only supports credentials through environment variables at this time. 
-
-**Linux/MacOS**
-```bash
-export AWS_ACCESS_KEY_ID=...
-export AWS_SECRET_ACCESS_KEY=...
-export AWS_REGION=... # eg. us-east-1
-```
-**Windows**
-```cmd
-set AWS_ACCESS_KEY_ID=...
-set AWS_SECRET_ACCESS_KEY=...
-set AWS_REGION=... # eg. us-east-1
-```
+
+3. Provide your AWS credentials with the default credential provider chain, which currently looks in:
+   - Environment variables: `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_REGION`
+   - Web Identity Token credentials from the environment or container
+   - The default credentials profile file, typically locaed in `~/.aws/credentials` (location can vary per platform)
```

Comment:
typo: located (can't make a commit suggestion because you can't make a suggestion on an individual commit)

Reply:
this isn't exactly right, we also support assume role credentials. from `~/.aws/config`